client/my-sites/stats/style.scss

```diff
@@ -27,6 +27,11 @@ $font-sf-pro-display: "SF Pro Display", $sans;
 	font-family: $font-sf-pro-display;
 }
 
+// App Promo card
+.is-odyssey-stats .app-promo-card__title {
```

Comment:
It looks like we already had some [Odyssey Stats font adjustment](https://github.com/Automattic/wp-calypso/blob/ce6f1b11c4d1fea35461de68ee7bc104107a95d1/apps/odyssey-stats/src/styles/typography.scss#L4-L10). Could we move these [.is-odyssey-stats weighted class names](https://github.com/Automattic/wp-calypso/blob/ce6f1b11c4d1fea35461de68ee7bc104107a95d1/client/my-sites/stats/style.scss#L12-L28) under the `apps/odyssey-stats/src/styles` folder to consolidate with the [mixin](https://github.com/Automattic/wp-calypso/blob/ce6f1b11c4d1fea35461de68ee7bc104107a95d1/apps/odyssey-stats/src/styles/variables.scss#L9) `stats-section-header-for-jetpack`? cc @kangzj

Reply:
Thanks for the pointer here @dognose24. This sounds like a nice improvement.

I'll ship these changes and then create a follow-up Issue for moving (and testing) the styles as part of the Odyssey app typography overrides. I have a few more font-related fixes to tackle with the UTM modal so I'll test this approach there.

Reply:
I tried this approach when updating the UTM Builder in https://github.com/Automattic/wp-calypso/pull/101357. Does this match what you had in mind @dognose24?

Reply:
I think so, except for the extra class name `is-odyssey-stats`, which could be replaced with another higher-hierarchy class name as it would be redundant on the Odyssey Stats build.